Wikipedia/Code/DiffContainerViewController.swift

```diff
@@ -38,21 +38,17 @@ class DiffContainerViewController: ViewController, HintPresenting {
 
     var hintController: HintController?
     
-    init(articleTitle: String, siteURL: URL, type: DiffContainerViewModel.DiffType, fromModel: WMFPageHistoryRevision?, toModel: WMFPageHistoryRevision, theme: Theme, diffController: DiffController? = nil, revisionDelegate: DiffRevisionRetrieving?) {
+    init(articleTitle: String, siteURL: URL, type: DiffContainerViewModel.DiffType, fromModel: WMFPageHistoryRevision?, toModel: WMFPageHistoryRevision, pageHistoryFetcher: PageHistoryFetcher? = nil, theme: Theme, revisionDelegate: DiffRevisionRetrieving?) {
         self.type = type
         
         self.fromModel = fromModel
         self.toModel = toModel
         self.articleTitle = articleTitle
         self.revisionDelegate = revisionDelegate
         self.siteURL = siteURL
-        
-        if let diffController = diffController {
-            self.diffController = diffController
```

Comment:
diffController wasn't ever passed in the init but feel free to change it if it was for future work